libs/langchain/Dockerfile

```diff
@@ -37,7 +43,7 @@ RUN $POETRY_HOME/bin/poetry install --no-interaction --no-ansi --with test
 FROM dependencies AS tests
 
 # Copy the rest of the app source code (this layer will be invalidated and rebuilt whenever the source code changes)
-COPY . .
+COPY langchain .
```

Comment:
this seems to assume running from parent dir. Could you try running from langchain dir?